pkg/selector/outputs/outputs.go

```diff
@@ -26,6 +26,20 @@ import (
 	"github.com/aws/amazon-ec2-instance-selector/v2/pkg/instancetypes"
 )
 
+// TruncateResults is used to prepare a list of details for output by truncating the number of results
+// in the list to have at most maxResults elements. Returns the truncated list of instance types and
+// the number of truncated items.
+func TruncateResults(maxResults *int, instanceTypeInfoSlice []*instancetypes.Details) ([]*instancetypes.Details, int) {
+	if maxResults == nil || *maxResults < 0 {
```

Comment:
Should we check for `maxResults = 0` as well?

Reply:
Oh yea good idea. I had thought that 0 wouldn't break the function, but thinking about it again, there would be no purpose to truncating all results and returning an empty list. I'll change `<` to `<=`.